### PR TITLE
Refactoring and standardizing modals

### PR DESCRIPTION
**Pull request by nidzix**

### CODE REVIEW

**Comment by ioanpocol** [comment]
:+1:

File: client/app/docs/module.js [hunk 1]
```diff
@@ -32,6 +43,23 @@
         return {
             restrict: 'C',
             link: function postLink(scope, element, attrs) {
+
+                //remove leading whitespaces
```
Comment (petrjasek) [inline client/app/docs/module.js]: you can use `String.trimLeft` for this

Comment (nidzix) [inline client/app/docs/module.js]: This is all about proper indentation of code, where every nested level should be indented 4chars more than previous one. So the idea is to pick indentation from first line, and trim left all other lines with that indentation.

Comment (petrjasek) [inline client/app/docs/module.js]: I see, my bad

**Comment by petrjasek** [comment]
:+1:

**Comment by amagdas** [comment]
:+1:

**Pull request merged**